site/content/community/channels/mailing-list.md

```diff
@@ -0,0 +1,12 @@
++++
+title = "Mailing List"
+weight = 20
+
+[asset]
+  icon = "fas fa-envelope-square"
+  url = "https://accounts.eclipse.org/mailing-list/cf-dev"
++++
+
+An excellent way to get the latest news and announcements around Californium is to join the mailing list.
+
+This is the place where we announce new releases and discuss architectural issues of greater impact.
```

Comment:
I'm not sure the mailing is so much used ?

Reply:
Would you like to change the wording only or do you want to remove the mailing list altogether?

Reply:
My idea : if nobody use it, we can remove it.

But If you think, we must keep it. It's OK for me too.

Reply:
Yes, it's really rare used.
I would keep it, for those, who don't have a github account (even if they area also rare :-) .
Explaining, that github issues (or eclipse tickets) are usually better for focus on discussing a topic.